mmdet/models/utils/misc.py

```diff
@@ -535,3 +535,80 @@ def get_box_tensor(boxes: Union[Tensor, BaseBoxes]) -> Tensor:
     if isinstance(boxes, BaseBoxes):
         boxes = boxes.tensor
     return boxes
+
+
+def filter_instances_by_score(batch_data_samples: SampleList,
```

Comment:
filter_gt_instances_by_score

Reply:
we can unify filter_gt_instances_by_score and filter_gt_instances_by_size through
`filter_gt_instances(batch_data_samples, score_thr=None, wh_thr=None)`